packages/e2e-tests/specs/editor/various/validate-multiple-use.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, insertBlock } from '@wordpress/e2e-test-utils';
+
+describe( 'Validate multiple use', () => {
```

Comment:
Could we move this test to [use Playwright](https://github.com/WordPress/gutenberg/blob/0ef7bc454ae1f9d32beb6bd40f2a4cb9d82a08b3/docs/contributors/code/testing-overview.md?plain=1#L514)?

Reply:
Hi @getdave, thanks for your review. I'm away from work for the next few weeks, so It would be nice if anyone were interested in taking this issue.